features/conversation-list/use-conversation-list-conversations.tsx

```diff
@@ -12,7 +12,11 @@ import { useEffect, useMemo } from "react";
 export const useConversationListConversations = () => {
   const currentAccount = useCurrentAccount();
 
-  const { data: conversations, ...rest } = useQuery(
```

Comment:
...rest destructuring causes excessive rerendering and potential performance issues. eslint rule prevents it now